.gitlab-ci.yml

```diff
@@ -152,6 +154,55 @@ test:kover:
     - python3 ddcoverage.py --prefix dd-sdk-android
     - bash <(cat ./codecov.sh) -t $CODECOV_TOKEN
 
+
+# TEST PYRAMID
+# the steps in this section should reflect our test pyramid strategy
+
+test-pyramid:single-fit-logs:
+  tags: [ "runner:main" ]
+  image: $CI_IMAGE_DOCKER
+  stage: test
```

Comment:
should it be `test-pyramid` here and below? Or it should stay `test` until this stage is completely replaced by `test-pyramid`?